pyanaconda/storage/initialization.py

```diff
@@ -150,16 +149,26 @@ def initialize_storage(storage):
     if not conf.system._is_live_os and not any(d.protected for d in storage.devices):
         raise UnknownSourceDeviceError()
 
-    # kickstart uses all the disks
-    if flags.automatedInstall:
-        disk_select_proxy = STORAGE.get_proxy(DISK_SELECTION)
-        selected_disks = disk_select_proxy.SelectedDisks
-        ignored_disks = disk_select_proxy.IgnoredDisks
 
-        if not selected_disks:
-            selected_disks = [d.name for d in storage.disks if d.name not in ignored_disks]
-            disk_select_proxy.SetSelectedDisks(selected_disks)
-            log.debug("onlyuse is now: %s", ",".join(selected_disks))
+def select_all_disks_by_default(storage):
```

Comment:
Not sure about having `by_default` in the method name. It is more how the method is used not what the method is doing.

Reply:
If there are already some disks selected, this method will do nothing. So it will really select all disks by default. I think that eventually, we will be able to this in a better way, so consider this to be a temporary fix.

Reply:
If it is just temporary then I may close one eye. But I'm still convinced there should be better name used.